lib/documents/conventions.rb

```diff
@@ -7,11 +7,11 @@
 
 module RavenDB
   class DocumentConventions
-    MaxNumberOfRequestPerSession = 30
```

Comment:
`MAX_NUMBER_OF_REQUEST_PER_SESSION` is not a constant and user should be able to modify that. It's just populated with default value.

Reply:
In Ruby, everything that begins with an uppercase letter is a constant.

Reply:
In Ruby, everything that begins with an uppercase letter is a constant.